eth_abi/codec.py

```diff
@@ -0,0 +1,146 @@
+from typing import (
+    Any,
+    Iterable,
+    Tuple,
+)
+
+from eth_typing.abi import (
+    Decodable,
+    TypeStr,
+)
+from eth_utils import (
+    is_bytes,
+)
+
+from eth_abi.decoding import (
+    ContextFramesBytesIO,
+    TupleDecoder,
+)
+from eth_abi.encoding import (
+    TupleEncoder,
+)
+from eth_abi.exceptions import (
+    EncodingError,
+)
+from eth_abi.registry import (
+    ABIRegistry,
+    registry as default_registry,
+)
+
+
+class ABICodec():
+
```

Comment:
We don't usually leave a blank line like just after a class declaration statement.